### PR TITLE
fix(report): align table headings & columns left & right

### PR DESCRIPTION
**Pull request by wardpeet**
**Summary**
It aligns our columns & headings correctly.

This is how it looks:
https://codepen.io/wardpeet/full/zJyMGx

**Related Issues/PRs**
#6049

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -907,18 +907,23 @@ summary.lh-passed-audits-summary {
 }
 
 .lh-table-column--text,
+.lh-table-column--url,
+.lh-table-column--node {
+  text-align: left;
```
[the remark this answers is not in the document]
Comment (wardpeet) [reply lighthouse-core/report/html/report-styles.css]: Yes th center by default I could override the default th behavior

File: lighthouse-core/report/html/report-styles.css [hunk 2]
```diff
@@ -907,18 +907,23 @@ summary.lh-passed-audits-summary {
 }
 
 .lh-table-column--text,
+.lh-table-column--url,
+.lh-table-column--node {
```
[the remark this answers is not in the document]
Comment (wardpeet) [reply lighthouse-core/report/html/report-styles.css]: Will do

File: lighthouse-core/report/html/report-styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulirish) [reply lighthouse-core/report/html/report-styles.css]: sg. lets add a comment of `/* Looks unnecessary, but mostly for keeping the <th>s left-aligned */`

File: lighthouse-core/report/html/report-styles.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (wardpeet) [reply lighthouse-core/report/html/report-styles.css]: `Array.from(new Set($$('[class^="lh-table-column--"]').map(el => el.className)))`

- lh-table-column--thumbnail
- lh-table-column--url
- lh-table-column--bytes
- lh-table-column--timespanMs
- lh-table-column--ms
- lh-table-column--text
- lh-table-column--empty
- lh-table-column--code
- lh-table-column--node

[the remark this answers is not in the document]
Comment (exterkamp) [reply lighthouse-core/report/html/report-styles.css]: does `.lh-table-column--code` belong in this list?

**Pull request merged**